src/charm.py

```diff
@@ -438,7 +438,7 @@ def _additional_log_configs(self) -> List[Dict[str, Any]]:
                             {
                                 "targets": ["localhost"],
                                 "labels": {
-                                    "__path__": "/var/log/*log",
+                                    "__path__": "/var/log/**/*log",
```

Comment:
Thanks @cbartz!
- Can you please confirm this change works as expected?
- Did you encounter an official doc mentioning globbing? If yes, can you please add the link as a comment here?

I skimmed through
- https://grafana.com/docs/loki/latest/send-data/promtail/configuration/
- https://grafana.com/docs/agent/latest/static/configuration/create-config-file/

and didn't notice anything.

Reply:
Hi @sed-i, yes, I tested the change locally and was able to successfully view the Juju logs transferred to Loki.
![image](https://github.com/canonical/grafana-agent-operator/assets/4182921/0a72a516-a126-4d0b-9686-ffb80ac263dc)

Support for glob patterns is mentioned here: https://grafana.com/docs/loki/latest/send-data/promtail/configuration/#static_configs :
```
targets:
  - localhost

# Defines a file to scrape and an optional set of additional labels to apply to
# all streams defined by the files from __path__.
labels:
  # The path to load logs from. Can use glob patterns (e.g., /var/log/*.log).
  __path__: <string>

  # Used to exclude files from being loaded. Can also use glob patterns.
  __path_exclude__: <string>

  # Additional labels to assign to the logs
  [ <labelname>: <labelvalue> ... ]
```

Reply:
@sed-i Is this information enough for you to merge the PR?